.github/scripts/get_workflow_type.py

```diff
@@ -49,47 +48,35 @@ def is_exception_branch(branch: str) -> bool:
 
 
 def get_workflow_type(issue: Issue, username: str) -> str:
-    user_list = issue.get_comments()[0].body.split("\r\n")
     try:
-        run_option = issue.get_comments()[1].body.split("\r\n")[0]
-    except Exception as e:
-        run_option = "single"
-
-    if user_list[0] == "!":
-        # Use old runners for everyone
-        return WORKFLOW_TYPE_LABEL
-    elif user_list[1] == "*":
-        if run_option == WORKFLOW_TYPE_BOTH:
-            # Use ARC runners and old runners for everyone
-            return WORKFLOW_TYPE_BOTH
-        else:
-            # Use only ARC runners for everyone
-            return WORKFLOW_TYPE_RG
-    elif username in user_list:
-        if run_option == WORKFLOW_TYPE_BOTH:
-            # Use ARC runners and old runners for a specific user
-            return WORKFLOW_TYPE_BOTH
+        user_list = issue.get_comments()[0].body.split("\r\n")
```

Comment:
also, is the indexing correct here?

```suggestion
        user_list = issue.get_comments()[1].body.split("\r\n")
```

Reply:
No, the first block is "body".  After the body we have a list of comments.
![Uploading Screenshot 2024-06-12 at 10.00.22.png…]()